demos/tab-indicator.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
```

Comment:
This demo page needs some restructure

* don't put so much in the "hero" section. You can just put two tabs there showing hte default behavior and call it done
* Add some sections, with titles, below the hero section. I'm thinking: bar x sliding indicator, icon x fading indicator, bar x fading indicator, icon x sliding indicator (same icon on each tab)
* Also add sections for the customization: color, height, and corner radius

Reply:
Updated